openapiv3/OpenAPIv3.go

```diff
@@ -2483,6 +2416,44 @@ func NewNamedString(in interface{}, context *compiler.Context) (*NamedString, er
 	return x, compiler.NewErrorGroupOrNil(errors)
 }
 
+// NewNamedStringArray creates an object of type NamedStringArray if possible, returning an error if not.
+func NewNamedStringArray(in interface{}, context *compiler.Context) (*NamedStringArray, error) {
+	errors := make([]error, 0)
+	x := &NamedStringArray{}
+	m, ok := compiler.UnpackMap(in)
+	if !ok {
+		message := fmt.Sprintf("has unexpected value: %+v (%T)", in, in)
+		errors = append(errors, compiler.NewError(context, message))
```

Comment:
Suggestion: put a `return` in this conditional block. `return x, compiler.NewErrorGroupOrNil(errors)`.

This keeps the error flow indented but the following "green path" code path in the `else` at minimal indentation.

Context: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
this is generated code, I'd like to address it separately in issue #167